XamlControlsGallery/ControlExample.xaml

```diff
@@ -63,8 +63,9 @@
             <RowDefinition />
         </Grid.RowDefinitions>
 
-        <TextBlock Style="{ThemeResource SubtitleTextBlockStyle}" Margin="0,15" Text="{x:Bind HeaderText}" />
-
+        <TextBlock Style="{ThemeResource SubtitleTextBlockStyle}" Margin="0,16" Text="{x:Bind HeaderText}" />
+        <TextBlock Margin="0,40,0,4" Text="{x:Bind onClickedText, Mode=TwoWay}"/>
```

Comment:
This logic adds the "OnClickedText" textblock to every control example, even if it wasn't needed. This adds unnecessary elements to every single control example. 

This change should localize the TextBlock specifically to the pages that need it, similar to the CommandBarFlyout page.

Reply:
I did another PR which utilises the same control, I was thinking of doing more for basically all the pages that have have user controls. The change I had at the beginning, showing the textblock below the code snippet was local to the page, however in order to get it above, and below the header I needed to modify the whole ControlExample, unless there's another way to do it?

Reply:
This logic is specifically applicable only to particular examples. My recommendation would be to put the TextBlock within the ControlExample. For example, consider something like:

```
        <local:ControlExample HeaderText="A simple MenuBar" 
                              XamlSource="MenuBar\MenuBarSample1.txt">
            <StackPanel>
                <TextBlock Text="You clicked..." />
                <muxc:MenuBar>
                    ...
                </muxc:MenuBar>
            </StackPanel>
        </local:ControlExample>
```

Reply:
I can find numerous examples which would utilise this change

Reply:
I can find numerous examples which would utilise this change but I will do as you request

Reply:
Generally the APIs on ControlExample should be widely applicable to most/all examples. I agree that there are several places that could use this sort of text, but I don't think that most/all examples need the click handler text. Hence the suggestion above.

Additionally, the inclusion of another TextBlock is another UIElement in the tree whether the specific example required the element or not. This causes an unnecessary performance hit for most controls. 

The benefit to including this text in the ControlExample would be consistency and ease of use (which are good benefits!) but since the feature isn't generally required, I believe we should encapsulate the logic to the specific examples that need them.

Reply:
Agreed, it will add unnecessary overhead for the Controls that don't utilise it.